src/components/WindRose/WindRoseChart.tsx

```diff
@@ -0,0 +1,29 @@
+import { Chart, type ChartData } from 'react-windrose-chart';
+
+import { columns } from '@/components/WindRose/data';
+
+interface TimeSeriesChartProps {
+  chartData: ChartData[] | null;
```

Comment:
```suggestion
  chartData: ChartData[];
```
I would suggest moving this check up and render the `WindRoseChart` only if there is data to pass for this component. Instead of passing `null`, I would prefer to render this component only if `chartData` is not `null`.